exact_solvers/acoustics_demos.py

```diff
@@ -40,30 +40,38 @@ def decompose_q(p,u,K,rho):
     should also print the eigenvectors and the values w_1, w_2."""
     Z = np.sqrt(K*rho)
     fig, axes = plt.subplots(1,2,figsize=(8,4)) 
-    axes[0].arrow(0,0,-Z,1,head_width=0.05, head_length=0.1, color=colors[0])
-    axes[0].arrow(0,0,Z,1, head_width=0.05, head_length=0.1, color=colors[1])
+    axes[0].arrow(0,0,-Z,1,head_width=0.07, head_length=0.15, 
+                  color=colors[0],lw=3)
+    axes[0].arrow(0,0,Z,1, head_width=0.07, head_length=0.15, 
+                  color=colors[1],lw=3)
     l1 = axes[0].plot([],[],colors[0])
     l2 = axes[0].plot([],[],'-',color=colors[1])
     axes[0].set_xlim(-2,2)
     axes[0].set_ylim(-2,2)
     axes[0].set_aspect('equal')
-    axes[0].set_title('Eigenvectors')
+    axes[0].set_title('Eigenvectors in phase plane',fontsize=10)
     axes[0].legend(['$r_1$','$r_2$'],loc=3)
     axes[0].plot([0,0],[-2,2],'--k',alpha=0.2)
     axes[0].plot([-2,2],[0,0],'--k',alpha=0.2)
+    axes[0].set_xlabel('p')
+    axes[0].set_ylabel('u')
 
     
-    axes[1].plot([0,p],[0,u],'k',lw=3)    
+    axes[1].plot([0,p],[0,u],'k-',lw=3,markersize=8)
+    #axes[1].plot([p],[u],'ko',markersize=8)    
```

Comment:
I'm guessing you meant to eventually delete this line and the other one also commented out just below.

Reply:
Yes, thanks.  Now done.